### PR TITLE
Add Mahowald invariant example

### PR DESCRIPTION
**Pull request by aergus**
Here is an example that computes algebraic Mahowald invariants, showcasing `ResolutionHomomorphism` in particular.

### CODE REVIEW

**Comment by hoodmane** [comment]
Sorry for just leaving this here. We should probably merge this...

File: ext/examples/mahowald_invariant.rs [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+        for (s, _, t) in s_2_resolution
+            .iter_stem()
+            .filter(|&(s, _, t)| p_k_resolution.has_computed_bidegree(s, t - 1))
+        {
+            let t_bottom = t + k as i32 - 1;
+            let bottom_s_2_gens = s_2_resolution.module(s).number_of_gens_in_degree(t_bottom);
+            let minus_one_s_2_gens = s_2_resolution.module(s).number_of_gens_in_degree(t);
+            let t_p_k = t - 1;
+            let p_k_gens = p_k_resolution.module(s).number_of_gens_in_degree(t_p_k);
```
Comment (JoeyBF) [inline ext/examples/mahowald_invariant.rs:130]:
```suggestion
            let bottom_s_2_gens = s_2_resolution.number_of_gens_in_bidegree(s, t_bottom);
            let minus_one_s_2_gens = s_2_resolution.number_of_gens_in_bidegree(s, t);
            let t_p_k = t - 1;
            let p_k_gens = p_k_resolution.number_of_gens_in_bidegree(s, t_p_k);
```

**Review by JoeyBF — APPROVED** [review]
It would be nice to have some tests, but I'm ok without too. Thanks for contributing!

**Comment by JoeyBF** [comment]
I'll add that the lint error is just due to a new version of clippy and the selenium error is the same as in #121

**Comment by aergus** [comment]
Thanks for the feedback. I'm trying to refactor the code to make it more testable and will comment here again when it's done (or I give up).

**Comment by aergus** [comment]
I've added a test with a few cases (and "fixed" the incompatibility with the `nassau` feature).

**Comment by JoeyBF** [comment]
Looks great to me! @hoodmane anything to add?

**Pull request merged**